geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/internal/cli/DestroyConnectionFunctionTest.java

```diff
@@ -83,13 +83,24 @@ public void getIdReturnsNameOfClass() throws Exception {
   }
 
   @Test
-  public void executeReportsErrorIfConnectionConfigNotFound() throws Exception {
-    function.execute(context);
+  public void serializes() throws Exception {
+    Serializable original = function;
 
-    ArgumentCaptor<CliFunctionResult> argument = ArgumentCaptor.forClass(CliFunctionResult.class);
-    verify(resultSender, times(1)).lastResult(argument.capture());
-    assertThat(argument.getValue().getErrorMessage())
-        .contains("Connection named \"" + connectionName + "\" not found");
+    Object copy = SerializationUtils.clone(original);
+
+    assertThat(copy).isNotSameAs(original).isInstanceOf(DestroyConnectionFunction.class);
+  }
+
+  @Test
+  public void destroyConnectionConfigReturnsTrueIfConnectionDestroyed() throws Exception {
+    when(service.getConnectionConfig(eq(connectionName))).thenReturn(configuration);
+
+    assertThat(function.destroyConnectionConfig(service, connectionName)).isTrue();
+  }
+
+  @Test
+  public void destroyConnectionConfigReturnsTrueIfConnectionDoesNotExist() throws Exception {
```

Comment:
Method name should be destroyConnectionConfigReturnsFalseIfConnectionDoesNotExist